### PR TITLE
Master

### PR DESCRIPTION
**Pull request by chengp3**
i18n support for lesson details pages

### CODE REVIEW

File: pages/lessons/[loc]/[id].js [hunk 1]
```diff
@@ -39,12 +40,15 @@ const LessonDetails = ({ lesson }) => {
     <Layout>
       <div className="bg-light-gray p-4">
         <div className="container">
-          {lastSubRelease && (
-            <p>
-              Version {lastSubRelease.version}{' '}
-              (Updated {format(new Date(lastSubRelease.date), 'MMM d, yyyy')})
-            </p>
-          )}
+          <div style={{display: "flex", justifyContent: "space-between"}}>
```
Comment (kenziebottoms) [inline pages/lessons/[loc]/[id].js:43]:
```suggestion
          <div className="d-flex justify-content-between">
```

**Review by kenziebottoms — CHANGES_REQUESTED** [review]
This is amazing!!!! Thank you!!!!

How do you feel about using `/lessons/1/en-us` instead of `/lessons/en-us/1`? I think that makes more sense structurally.

File: pages/lessons/[loc]/[id].js (hunk 1 above)
Comment (kenziebottoms) [reply pages/lessons/[loc]/[id].js:43]: we also need to redirect `/lessons/1` to the default translation

**Comment by chengp3** [comment]
Absolutely yes I agree but unfortunately the i18n module in react doesn’t
create urls in that form afaik?

**Comment by kenziebottoms** [comment]
> Absolutely yes I agree but unfortunately the i18n module in react doesn’t create urls in that form afaik?


What do you mean? It looks to me like you're creating the URLs in that format in `LocDropdown.js#22` and by changing that, and moving `pages/lessons/[loc]/[id].js` to `pages/lessons/[id]/[loc].js` i think we can swap params.

**Pull request closed without merging**